specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -3439,28 +3594,57 @@
     "ReservationTransaction": {
       "description": "Reservation transaction resource.",
       "type": "object",
+      "discriminator": "kind",
```

Comment:
>"discriminator": "kind", [](start = 6, length = 24)

these kinds of changes to an existing api-version will break clients.  Please introduce a new api-version

Reply:
The API only supports legacy accounts currently. The only change for the existing callers is an additional "kind" property. 

With this change, callers with modern accounts will be able to use the API.

Adding a new api-Version is an expensive change on our side, which we are planning to do some time later.

Reply:
What happens when a user with a modern account tries to use this API with existing clients that don't support this new contract?